server/views/partials/analytics/chartTrends.njk

```diff
@@ -9,7 +9,7 @@
   form,
   csrfToken,
   report,
-  showPrisonPopulation=true,
+  showTotalPopulation=true,
```

Comment:
Renamed this because for this chart we show total population for the PC, not the prison